Chargebee/Classes/Restore/CBRestorePurchaseManager.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  CBRestorePurchaseManager.swift
+//  Chargebee
+//
+//  Created by ramesh_g on 20/02/23.
+//
+
+import Foundation
+import StoreKit
+
+public typealias CBRestorePurchaseHandler = (CBResult<CBRestorePurchase>) -> Void
+
+// MARK: - CBRestorePurchase
+public struct CBRestorePurchase: Decodable {
+    let inAppSubscriptions: [InAppSubscription]
+
+    enum CodingKeys: String, CodingKey {
+        case inAppSubscriptions = "in_app_subscriptions"
+    }
+}
+
+// MARK: - InAppSubscription
+public struct InAppSubscription: Decodable {
+    public let subscriptionID, planID, storeStatus, purchaseType: String?
```

Comment:
- Are all of these values optional? Lets make sure we map the values accordingly.
- Should `storeStatus` and `purchaseType` be strings? Can we enumerate them accordingly?

Reply:
All are non optionals only and `purchaseType ` we removed because its not there in API contract, and storeStatus i created enum to enumerate .